proto/axelar/nexus/exported/v1beta1/types.proto

```diff
@@ -96,6 +96,8 @@ message GeneralMessage {
     STATUS_UNSPECIFIED = 0 [ (gogoproto.enumvalue_customname) = "NonExistent" ];
     STATUS_APPROVED = 1 [ (gogoproto.enumvalue_customname) = "Approved" ];
     STATUS_EXECUTED = 2 [ (gogoproto.enumvalue_customname) = "Executed" ];
+    STATUS_FAILED = 3 [ (gogoproto.enumvalue_customname) = "Failed" ];
+    STATUS_WAITING = 4 [ (gogoproto.enumvalue_customname) = "Waiting" ];
```

Comment:
What's this for?

Reply:
this is gone now, was just a placeholder for some sort of pending status